CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add `UploadedAttachmentPostProcessor` in `ChatClientConfig` to allow changing custom attachment payloads after an attachment has been uploaded [#2457](https://github.com/GetStream/stream-chat-swift/pull/2457)
 - Add `AnyAttachmentPayload(localFileURL:customPayload:)` initializer to allow creating custom attachments without a remote URL [#2457](https://github.com/GetStream/stream-chat-swift/pull/2457)
 - Add skip push support when sending a message [#2486](https://github.com/GetStream/stream-chat-swift/pull/2486)
+- Add `isChannelLocalFilteringEnabled` in `ChatClientConfig` to allow changing whether the Channels in ChannelList will be locally filtered [#2488](https://github.com/GetStream/stream-chat-swift/pull/2488)
```

Comment:
I think that it also makes sense to add a point, to just mention that now, channels will be automatically filtered:
```suggestion
- Add support for automatically filtering channels in the Channel List [#2488](https://github.com/GetStream/stream-chat-swift/pull/2488)
- Add `isChannelLocalFilteringEnabled` in `ChatClientConfig` to allow changing whether the Channels in ChannelList will be locally filtered [#2488](https://github.com/GetStream/stream-chat-swift/pull/2488)
```